### PR TITLE
build: Arch User Repository

### PR DESCRIPTION
**Pull request by morrieinmaas**
First simple PKGBUILD added

See commit for TODOS

### CODE REVIEW

File: docs/features/environments.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ configurations:
     token: ~
```
Comment (morrieinmaas) [inline docs/features/environments.md:19]: This change will be removed - sorry that was my linter/auto-formatting @blu3beri

**Comment by blu3beri** [comment]
Any reason why `/usr/local/share`? And what if the dir does not exist?

**Comment by blu3beri** [comment]
I am not too sure but so we have write access to /usr/local/bin?

**Comment by morrieinmaas** [comment]
Just first draft @blu3beri. will need to test this and spin up an arch machine for this

**Comment by morrieinmaas** [comment]
Yeah I'm also not sure. Otherwise /usr/bin is where binaries live if I remember correctly. I'll give that a try tomorrow. Open to and happy for suggestions if you know the way to go here

**Comment by blu3beri** [comment]
> Yeah I'm also not sure. Otherwise /usr/bin is where binaries live if I remember correctly. I'll give that a try tomorrow. Open to and happy for suggestions if you know the way to go here

`/usr/bin` is protected for sure. However yay 99% requires sudo so we might just have to curl it to `tmp` and sudo mv it. Just guessing here. I think I would prefer `/usr/local/bin` but we should just follow aur conventions ;).

**Comment by morrieinmaas** [comment]
sounds good to me. I'll give it a go

File: PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+provides=('aries-cli')
+md5sums=()
+
+package() {
+  sudo mkdir -p /usr/share/licenses/${pkgname}
+  sudo curl -L -o /usr/share/licenses/${pkgname}/LICENSE https://raw.githubusercontent.com/animo/aries-cli/main/LICENSE
+  echo "Fetching aries-cli binary"
+  sudo curl -L -o /usr/bin/${pkgname} https://github.com/animo/${pkgname}/releases/download/v${pkgver}/linux-x86_64-aries-cli
+  sudo chmod +x /usr/bin/${pkgname}
+  echo "Successfully installed aries-cli"
```
Comment (blu3beri) [inline PKGBUILD:18]: Id remove the echo and keep it just to the install. If it works I will accept it.

**Review by blu3beri — CHANGES_REQUESTED** [review]

File: docs/features/environments.md [hunk 3]
```diff
@@ -19,7 +19,9 @@ configurations:
...
 ```
 
-
+{% hint style="warning" %}
```
Comment (blu3beri) [inline docs/features/environments.md:22]: Remove this commit from your pr :d

Comment (morrieinmaas) [reply docs/features/environments.md:22]: ok nice - will remove it. tested it today and install worked with yay

Comment (morrieinmaas) [reply docs/features/environments.md:22]: i'll make a pipeline for auto release AUR on release of new version in a separate PR then. started on it already

Comment (morrieinmaas) [reply docs/features/environments.md:22]: @blu3beri aaaand changes puuuuuuuushT

File: PKGBUILD (hunk 2 above)
Comment (morrieinmaas) [reply PKGBUILD]: done

File: docs/features/environments.md (hunk 3 above)
Comment (morrieinmaas) [reply docs/features/environments.md:22]: ah damn sorry this one - will do- lol

**Comment by morrieinmaas** [comment]
@blu3beri alright should be good to go now

**Review by blu3beri — CHANGES_REQUESTED** [review]
Great work!

Just one small thing, can we create a folder `pkg` with a folder `aur` and put it in there? Then we can just manage it from there for all the other repos :)

**Comment by morrieinmaas** [comment]
thanks. and of course. let me push that

**Comment by blu3beri** [comment]
> thanks. and of course. let me push that

**Comment by morrieinmaas** [comment]
@blu3beri done

**Review by blu3beri — APPROVED** [review]

**Pull request merged**